### ISSUE TITLE
Suggestion: items don't apply vanilla effects

### ISSUE THREAD

**Issue by Jackiecrazy** [ElColoManco/Risk-of-Rain-Mod#7]
As vanilla effects don't stack, it's not possible to use, say, bustling fungus with regeneration potions, or soldier's syringe near a haste beacon. It would be great to have item effects get calculated separately so they can stack in those situations.

**Comment by ElColoManco** [comment]
Yes, It was the first thing I thought of when creating the first item (fungus). I still haven't found the way to stack effects. But I release them that way anyway because vanilla will take the highest amplifier of the same effect. Say you have just 1 fungus and a regen2 potion, the potion will have priority when applied

**Comment by Jackiecrazy** [comment]
I wrote a helper function to stack potion effects a while ago, I can send it to you if you need it. Another way would be to use a capability for the fungus tick and directly modify attributes for soldier's syringe and goat hoof. Rose buckler is quite powerful, so non-stacking makes sense.
